source/tdx-guest-hardening.rst

```diff
@@ -1000,4 +1000,325 @@ https://github.com/intel/ccc-linux-guest-hardening/tree/master/bkc/kafl/userspac
 More sophisticated “harness” for randomized stimulus execution:
 https://github.com/intel/ccc-linux-guest-hardening/tree/master/bkc/kafl/userspace/sharedir_template/init.sh
 
+Enabling additional kernel drivers
+==================================
 
+The reference TDX guest kernel implementation provided for the `Linux SW stack for
+Intel TDX <https://github.com/intel/tdx-tools>`_ only enables a small set of
+virtio drivers that are essential for the TDX guest basic functionality. These
+drivers have been hardened using the methodology described in this document,
+but naturally different deployment scenarios and use cases for the TDX will
+require many more additional drivers to be enabled in the TDX guest kernel.
+
+This section provides guidance on how to use the methodology presented
+in this document for adding and hardening a new driver for the TDX guest kernel.
+
+Step-by-step guidance
```

Comment:
You are declaring only one section here, and no other section in the rest of this chapter.
This ends up with a strange hierarchy:
![image](https://user-images.githubusercontent.com/964610/178297223-7b148721-b226-4095-8e92-613d39f406cf.png)

Maybe you could instead use each step as a separate section ?